src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php

```diff
@@ -160,7 +159,6 @@ private function decideConsensus(TokenInterface $token, array $attributes, $obje
                     break;
 
                 default:
```

Comment:
do we have other empty "default" block in the code base? if not, I'd suggest removing it here also